code/drasil-printers/lib/Language/Drasil/TeX/Print.hs

```diff
@@ -345,17 +357,17 @@ makeDefTable sm ps l = mkEnvArgBr "tabular" (col rr colAwidth ++ col (rr ++ "\\a
 
 -- | Helper that makes the rows of a definition table.
 makeDRows :: PrintingInformation -> [(String,[LayoutObj])] -> D
-makeDRows _  []         = error "No fields to create Defn table"
-makeDRows sm ls    = foldl1 (%%) $ map (\(f, d) -> dBoilerplate %%  pure (text (f ++ " & ")) <> print sm d) ls
+makeDRows _  []      = error "No fields to create Defn table"
+makeDRows sm ls      = foldl1 (%%) $ map (\(f, d) -> dBoilerplate %%  pure (text (f ++ " & ")) <> print' sm d) ls
   where dBoilerplate = pure $ dbs <+> text "\\midrule"
```

Comment:
Is `dBoilerplate` reused anywhere? If not, it might make more sense to just inline it, but others may disagree

Reply:
Fixed in https://github.com/JacquesCarette/Drasil/pull/3910/commits/f78f58a2d6d93b74e7b82afe12e536ffedbacf8c.